src/api/services/Overpass.ts

```diff
@@ -25,4 +26,30 @@ export default {
       { enabled: !!box, refetchOnWindowFocus: false },
     );
   },
+  useNearestPlaces: (cityName: string, coordinates?: string) => {
+    const query = `[out:json][timeout:25];
+    (
+      node["place"~"town|city"](around:50000,${coordinates});
+    );
+    out body 7;
+    >;
+    out skel qt;`;
+
+    return useQuery<Place[], Error>(
+      ['cities', cityName],
+      async () => {
+        const res = await overpassClient.post('/interpreter', query);
+
+        const normalizedNearestPlaces = res.data.elements
+          .map((city: Place) => {
+            return { ...city, tags: { name: convertSpecialCharacters(city.tags.name) } };
+          })
+          .filter((city: Place) => city.tags.name !== cityName)
+          .slice(0, 6);
```

Comment:
**Question** Is this line necessary? If we filter the cities in the line above we should get 6 places anyways

Reply:
Unfortunately, sometimes this city is on the list, and sometimes not, so for example in the case of Wrocław there are 7 cities on the list

Reply:
Oh, okay, I get it